src/main/app/case/definition.ts

```diff
@@ -374,6 +375,10 @@ export const enum JurisdictionConnections {
    * Applicant 2 is domiciled in England and Wales
    */
   APP_2_DOMICILED = 'I',
+  /**
+   * Applicant 1 habitually resides in England and Wales and joint application
+   */
+  APP_1_RESIDENT = 'J',
```

Comment:
```suggestion
  APP_1_RESIDENT_JOINT = 'J',
```

Reply:
Yeah that's a better label, will need to update on backend first though